### ISSUE TITLE
Add text alternate to social media icons (accessibility fix)

### ISSUE THREAD

**Issue by Roenok** [girldevelopit/gdi-website#344]
**Description**
The Facebook and Twitter icons in the header and footer of the site are empty links with a Font Awesome icon. Because they have no text, they are invisible to screen reader users. When you tab over them, the reader just says "link"

**Ask**
We should add some hidden text to these links for screen reader users. It will also be a fallback if our CSS fails to load for some reason. It's a pretty simple fix, with two parts:

_Add a CSS class for screen reader only things_

``` css
.sr-only {
  font-size: 0;
  height: 1px;
  overflow: hidden;
  display: block;
}
```

Note this is clipped, not moved off screen or hidden. Off-screen text would do weird things when navigating with a keyboard, and display:none or visibility:hidden would hide the code from screen reader users too.

_Add hidden text to the HTML_

``` html
<a target="_blank" href="https://www.facebook.com/girldevelopit">
  <i class="fa fa-facebook"><span class="sr-only">GDI Facebook</span></i>
</a>
```

**To Do**
- [ ] Add CSS class
- [ ] Modify header and footer
- [ ] Validate with a screen reader

**Additional Info**
Yay accessibility!

**Comment by AishaBlake** [comment]
Submitted a pull request! Instead of creating a class, I added `aria-label` attributes to each icon. This is my first time contributing. Is there anything else I need to include here?

**Comment by Roenok** [comment]
Thanks! An aria-label attribute is also a good fallback. We'll review this during our next set of commits. 

As far as submitting, it is helpful to link your pull request to the issue (i.e., say fixes #issue) to help us know what to close. I added that as a comment to the pull request.

**Comment by AishaBlake** [comment]
Thank you!! I'll be sure to do that next time!
